spec/presenters/section_presenter_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+describe SectionPresenter do
+  let(:section) { double(:section, body: '', attributes: {}, attachments: []) }
```

Comment:
Minor: Unless there's a strong reason not to I'd be tempted to use a real `Section` instance instead of a mock.